src/plone/api/tests/test_group.py

```diff
@@ -480,17 +481,27 @@ def test_grant_roles_in_context(self):
         ROLES = set(['Editor', 'Contributor'])
         self.assertEqual(
             ROLES,
-            set(api.group.get_roles(
-                groupname='foo', obj=folder, inherit=False)),
+            set(
+                api.group.get_roles(
+                    groupname='foo',
+                    obj=folder,
+                    inherit=False,
+                ),
+            ),
         )
         self.assertEqual(
             ROLES,
-            set(api.group.get_roles(group=group, obj=folder, inherit=False)),
+            set(api.group.get_roles(group=group, obj=folder, inherit=False,)),
```

Comment:
Sorry for being late on the party (I've been really busy at work).

I could be ok with not enforcing a comma in this specific case, but the statement is *broken* already by having a statement with multiple parameters as a second argument. To sum it up: http://treyhunner.com/2017/07/craft-your-python-like-poetry/

I would change that to:
```
expected_roles = api.group.get_roles(
    group=group,
    obj=folder,
    inherit=False,
)
self.assertEqual(
    ROLES,
    set(expected_roles),
)
```

This is, by far, more readable and has no problem. Readability counts, not cramming as much as you can on a single line of text 😃